### PR TITLE
Add checkbox for adding a time offset to lookback time ranges

### PR DESCRIPTION
**Pull request by zjanc**
# Add checkbox for adding a time offset to lookback time ranges

## Context

Jaeger typically surfaces incomplete traces when a user searches with the default `lookback` option. To improve the general user experience of first time users, we add a default offset of 5 minutes to the `lookback` to surface more complete traces.

### CODE REVIEW

File: packages/jaeger-ui/src/components/SearchTracePage/SearchForm.js [hunk 1]
```diff
@@ -369,6 +374,16 @@ export class SearchFormImpl extends React.PureComponent {
           </Field>
         </FormItem>
 
+        {selectedLookback !== 'custom' && (
```
Comment (jaskerv) [inline packages/jaeger-ui/src/components/SearchTracePage/SearchForm.js:377]: I have a suggestion for this part of the UI. 
My demo here:
https://codesandbox.io/s/basic-antd-4-22-6-forked-301xpo?file=/demo.js:270-899

File: packages/jaeger-ui/src/components/SearchTracePage/SearchForm.js [hunk 2]
```diff
@@ -369,6 +374,16 @@ export class SearchFormImpl extends React.PureComponent {
...
+          <FormItem label="Lookback Offset" className="checkbox--LookbackOffset">
+            <Row gutter={16}>
+              <Col span={4}><Field name="offsetEnabled" component={AdaptedCheckbox} defaultChecked={offsetEnabled} type="checkbox" /></Col>
+              <Col span={20}>Add a 5 minute offset to the search to ensure you only see complete traces.</Col>
+            </Row>
+          </FormItem>
+        )}
+
```
Comment (jaskerv) [inline packages/jaeger-ui/src/components/SearchTracePage/SearchForm.js:405]: Nit: Extra white space.

**Review by jaskerv — APPROVED** [review]
Looks great :D I have a suggestion for the UI. Otherwise LGTM

File: packages/jaeger-ui/src/components/SearchTracePage/SearchForm.js (hunk 1 above)
Comment (zjanc) [reply packages/jaeger-ui/src/components/SearchTracePage/SearchForm.js:377]: Used a popover instead similar to what's used for the `Tag` field, also avoids importing a new package.

**Comment by zjanc** [comment]
Jaeger Lookback Offset switch with popover:

Jaeger Lookback Offset

**Pull request merged**